js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -24,37 +27,68 @@ import PaidAdsFeaturesSection from './paid-ads-features-section';
 import GoogleAdsAccountSection from './google-ads-account-section';
 import AudienceSection from '.~/components/paid-ads/audience-section';
 import BudgetSection from '.~/components/paid-ads/budget-section';
+import BillingCard from '.~/components/paid-ads/billing-card';
 import validateForm from '.~/utils/paid-ads/validateForm';
 import { getProductFeedUrl } from '.~/utils/urls';
-import { GUIDE_NAMES, GOOGLE_ADS_ACCOUNT_STATUS } from '.~/constants';
 import { API_NAMESPACE } from '.~/data/constants';
+import {
+	GUIDE_NAMES,
+	GOOGLE_ADS_ACCOUNT_STATUS,
+	GOOGLE_ADS_BILLING_STATUS,
+} from '.~/constants';
 
-const { CONNECTED } = GOOGLE_ADS_ACCOUNT_STATUS;
-
-function PaidAdsSectionsGroup( { onCampaignChange } ) {
+function PaidAdsSectionsGroup( { onStatesReceived } ) {
```

Comment:
💅  WDYT of moving this to a separate component inside `setup-paid-ads-folder`

Reply:
💅  WDYT of naming this component `CreatePaidAdsCampaign` ? I agree that this is a group of Sections related to Paid Ads... but `PaidAdsSectionsGroup` was not very clarifying for me in terms of what action we are doing and what those "Sections groups" are doing .

Reply:
To facilitate the new round of code reviews, I will merge the component extraction to this PR from the `tweak/1610-extract-paid-ads-setup-sections` branch after this PR gets approval. The diff can be viewed in this link - https://github.com/woocommerce/google-listings-and-ads/compare/update/1610-billing-and-campaign-setup...tweak/1610-extract-paid-ads-setup-sections